lib/manifest.ps1

```diff
@@ -44,24 +44,23 @@ function Get-Manifest($app) {
         if ($bucket) {
             $manifest = manifest $app $bucket
         } else {
-            foreach ($bucket in Get-LocalBucket) {
-                $manifest = manifest $app $bucket
+            foreach ($tekcub in Get-LocalBucket) {
```

Comment:
The `$tekcub` is? A random name?

Reply:
no, it's simply spelled backward of `bucket`.